lms/djangoapps/course_home_api/outline/v1/serializers.py

```diff
@@ -21,8 +21,35 @@ def get_url(self, tool):
         return request.build_absolute_uri(url)
 
 
+class CourseBlockSerializer(serializers.Serializer):
+    """
+    Serializer for Course Block Objects
+    """
+    blocks = serializers.SerializerMethodField()
+
+    def get_blocks(self, blocks):
+        data = {}
+
+        for block_key in blocks:
```

Comment:
Another way to do this chunk of code, since all you are doing is iterating and adding to a dict would be... (and this is all a nit, I just love comprehensions):

```
return {
  str(block_key): {
    'id': ...,
  }
  for block_key in blocks
}
```

That's a dicitonary comprehension. Whether that is prettier or more understandable is up to you.

Reply:
That's much cleaner!! Thanks for showing me that